mobile/src/main/java/ch/epfl/sdp/mobile/ui/home/HomeScaffold.kt

```diff
@@ -46,16 +46,19 @@ enum class HomeSection(
 fun HomeScaffold(
     section: HomeSection,
     onSectionChange: (HomeSection) -> Unit,
+    hideBar: Boolean,
```

Comment:
Nitpick : I'd name this `hiddenBottomBar` instead, because :

- there can be multiple bars (top bar, bottom bar)
- I feel like `hide` may indicate an action, and `hidden` describes a state

Reply:
Agree